devicesetup/src/main/res/layout/activity_password_entry.xml

```diff
@@ -38,23 +38,23 @@
                 android:id="@+id/ssid"
                 android:layout_width="0px"
                 android:layout_height="wrap_content"
-                android:layout_gravity="left"
```

Comment:
Using "start" was only added with API 17, IIRC, but the SDK still supports API 14.